Corona-Warn-App/src/test/java/de/rki/coronawarnapp/datadonation/OneTimePasswordTest.kt

```diff
@@ -0,0 +1,18 @@
+package de.rki.coronawarnapp.datadonation
+
+import io.kotest.matchers.shouldBe
+import okio.ByteString.Companion.decodeBase64
+import org.junit.Test
+import java.util.UUID
+
+class OneTimePasswordTest {
+
+    @Test
+    fun test() {
```

Comment:
A more descriptive test name would be appreciated 🙂

Reply:
renamed it